react-app/src/components/TopNavigationBar/index.js

```diff
@@ -1,9 +1,41 @@
-function TopNavigationBar() {
-    return (
-        <>
-            <h1>TopNavigation Bar</h1>
-        </>
-    );
+
+import React from "react";
+import { NavLink } from "react-router-dom";
+import { useSelector } from "react-redux";
+import ProfileButton from "./ProfileButton";
+import "./Navigation.css";
+
+
+function TopNavigation({ isLoaded }) {
```

Comment:
please rename function and export to TopNavigationBar to be consistent with folder name